spec/lib/myfinance/http_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe Myfinance::Http do
+  subject { Myfinance::Http.new('b552dd5a8598ca089b91c5e355a83b86e4696aefac9eaa05') }
+
+  describe '#get' do
+    let(:http_method) { :get }
+    let(:url) { '/accounts' }
+    let(:params) { {} }
+
+    it_behaves_like 'available http request methods'
+  end
+
+  describe '#post' do
+    let(:http_method) { :post }
+    let(:url) { '/entities' }
+    let(:params) { { entity: { name: 'Second entity' } } }
+
+    it_behaves_like 'available http request methods'
+  end
+
+  describe '#put' do
+    let(:http_method) { :put }
+    let(:url) { '/entities/3798' }
+    let(:params) { { entity: { name: 'Second updated entity' } } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.